requirements-base.txt

```diff
@@ -35,7 +35,9 @@ Pillow==8.2.0; python_version == '3.6'
 Pillow==8.3.1; python_version > '3.6'
 progressbar2==3.32.0
 python-rapidjson==1.4
-psycopg2-binary==2.8.6
+psycopg2-binary==2.8.6; python_version == '3.6'
+# This removes needing to set CPPFLAGS & LDFLAGS
+psycopg2-binary==2.9.1; python_version > '3.8'
```

Comment:
If I don't do this, I would have to do this somewhere in `install-py-dev`:

```shell
CPPFLAGS="-I/opt/homebrew/opt/openssl@1.1/include" \
    LDFLAGS="-L/opt/homebrew/opt/openssl@1.1/lib -L/opt/homebrew/opt/python@3.8/lib" \
    pip install psycopg2-binary==2.8.6
```

Reply:
You won't be installing this on 3.8.

```
Ignoring psycopg2-binary: markers 'python_version == "3.6"' don't match your environment
Ignoring psycopg2-binary: markers 'python_version > "3.8"' don't match your environment
```

Reply:
Good catch! It was a last second commit that introduced the problem. I've corrected this and tested it out.